cmd/tsidp/README.md

```diff
@@ -12,38 +12,64 @@
 
 ## Installation using Docker
 
-1. **Build the Docker Image**
+### Official Images (Recommended)
 
-   The Dockerfile uses a multi-stage build process to:
-   - Build the `tsidp` binary from source
-   - Create a minimal Alpine-based image with just the necessary components
+The recommended way to run `tsidp` is using the official pre-built images available at:
+- `tailscale/tsidp:latest` (or specific version tags)
+
+```bash
+# Run the official image
+docker run -d \
+  --name tsidp \
+  -p 443:443 \
+  -e TS_AUTHKEY=YOUR_TAILSCALE_AUTHKEY \
+  -v tsidp-data:/var/lib/tsidp \
+  tailscale/tsidp:latest \
+  --hostname=idp --dir=/var/lib/tsidp
+```
+
+### Building from Source
+
+1. **Using the Standard Build Script (Recommended)**
+
+   The Tailscale project uses `mkctr` for building container images:
 
    ```bash
    # Clone the Tailscale repository
    git clone https://github.com/tailscale/tailscale.git
    cd tailscale
+   
+   # Build the tsidp image
+   TARGET=tsidp ./build_docker.sh
    ```
 
+   For development, you can publish to your own registry:
+   ```bash
+   make publishdevtsidp REPO=ghcr.io/yourusername/tsidp
+   ```
+
+2. **Using the Dockerfile (Fallback)**
+
    ```bash
-   # Build the Docker image
-   docker build -t tsidp:latest -f cmd/tsidp/Dockerfile .
+   # Build using the Dockerfile directly
+   docker build -t tailscale/tsidp -f cmd/tsidp/Dockerfile .
    ```
 
-2. **Run the Container**
+3. **Run the Container**
 
-   Replace `YOUR_TAILSCALE_AUTHKEY` with your Tailscale authentication key.
+   Replace `YOUR_TAILSCALE_AUTHKEY` with your Tailscale authentication key:
 
    ```bash
    docker run -d \
      --name tsidp \
      -p 443:443 \
      -e TS_AUTHKEY=YOUR_TAILSCALE_AUTHKEY \
-     -e TS_HOSTNAME=idp \
      -v tsidp-data:/var/lib/tsidp \
-     tsidp:latest
+     tailscale/tsidp:latest \
+     --hostname=idp --dir=/var/lib/tsidp
```

Comment:
This will not work with images built with `mkctr` where you are not setting entrypoint in the same way as for the ones you built with Docker.
You need to do `... tailscale/tsidp:latest tsidp --hostname foo --dir /foo/bar`